ext/standard/basic_functions.stub.php

```diff
@@ -1778,15 +1787,15 @@ function array_intersect_uassoc(array $array, ...$rest): array {}
 function array_uintersect_uassoc(array $array, ...$rest): array {}
 
 /**
- * @refcount 1
  * @compile-time-eval
+ * @refcount 1
  */
 function array_diff_key(array $array, array ...$arrays): array {}
 
 /**
  * @param array|callable $rest
- * @refcount 1
  * @compile-time-eval
+ * @refcount 1
  */
 function array_diff_ukey(array $array, ...$rest): array {}
```

Comment:
https://github.com/php/php-src/pull/10842/files#r1137359284 - shouldn't the CTE flag be actually removed here?